### ISSUE TITLE
devcontainer: support docker-compose 

### ISSUE THREAD

**Issue by johnstcn** [coder/envbuilder#236]
Devcontainer spec supports specifying a compose file

e.g.

```
	"dockerComposeFile": "docker-compose.yml",
```

Example: https://github.com/BrunoQuaresma/openticket/blob/main/.devcontainer/devcontainer.json

**Comment by jasclarke24** [comment]
This would be an awesome feature. We run a lot of dev environments with multiple containers for testing and rapid integration. What is the recommended way of doing this now, multiple containers in the template?

**Comment by johnstcn** [comment]
> What is the recommended way of doing this now, multiple containers in the template?
Yeah that's honestly the best way right now. We're working on an alternative method of using devcontainer that should more seamlessly enable using `@devcontainers/cli` or similar existing Devcontainer tooling.

**Comment by tomtom5152** [comment]
I notice #107 and #135 both mention the same feature set, with the latter claiming it is a massive overhaul to add support for it. Is there still the plan to switch to the `@devcontainers/cli` tooling or something else? Happy to help out with this if I can, but some overall guidance on where the project wants to take it would be very helpful thanks
